packages/template-retail-react-app/app/pages/checkout/index.test.js

```diff
@@ -10,49 +10,50 @@ import {Route, Switch} from 'react-router-dom'
 import {screen, waitFor, within} from '@testing-library/react'
 import user from '@testing-library/user-event'
 import {rest} from 'msw'
-import {renderWithProviders, createPathWithDefaults} from '../../utils/test-utils'
-import useShopper from '../../commerce-api/hooks/useShopper'
-import Auth from '../../commerce-api/auth'
 import {
-    ocapiBasketWithItem,
-    ocapiOrderResponse,
+    renderWithProviders,
+    createPathWithDefaults,
+    registerUserToken
+} from '../../utils/test-utils'
+import {
+    scapiBasketWithItem,
     mockShippingMethods,
-    mockPaymentMethods,
-    mockedGuestCustomer,
     mockedRegisteredCustomer,
-    mockedCustomerProductLists,
-    productsResponse
+    mockedCustomerProductLists
 } from '../../mocks/mock-data'
 import mockConfig from '../../../config/mocks/default'
 
-jest.mock('commerce-sdk-react-preview', () => {
-    const originalModule = jest.requireActual('commerce-sdk-react-preview')
-    return {
-        ...originalModule,
-        useCustomerBaskets: jest
-            .fn()
-            .mockReturnValue({data: {baskets: [{shippingItems: [{}], currency: 'GBP'}]}})
-    }
-})
+jest.setTimeout(30000)
 
-// Make sure fetch is defined in test env
-Object.defineProperty(window, 'fetch', {
-    value: require('cross-fetch')
-})
+// Minimal subset of `ocapiOrderResponse` in app/mocks/mock-data.js
+const scapiOrderResponse = {
+    orderNo: '00000101',
+    customerInfo: {
+        customerId: 'customerid',
+        customerNo: 'jlebowski',
+        email: 'jeff@lebowski.com'
+    }
+}
 
-const {keysToCamel} = jest.requireActual('../../commerce-api/utils')
```

Comment:
This `keysToCamel` was no longer needed, as we now have switched fully to SCAPI (not mixing both OCAPI and SCAPI).